{"uuid": "abf78fe9-e215-485b-9fb5-6c0d8392373c", "children": ["c87733a6-79a3-4ab3-aa7f-0d3bfb3e2fc0"], "befores": [{"name": "country", "status": "broken", "statusDetails": {"message": "AttributeError: 'function' object has no attribute 'get'\n", "trace": "  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\Tests\\conftest.py\", line 121, in country\n    country.get(\"https://cosmocode.io/automation-practice-webtable/\")\n    ^^^^^^^^^^^\n"}, "start": 1730824034228, "stop": 1730824034228}], "start": 1730824034228, "stop": 1730824034393}